gcovr/configuration.py

```diff
@@ -623,6 +623,15 @@ def choose(_cls, choices, default=None):
         default=None,
         const=OutputOrDefault(None),
     ),
+    GcovrConfigOption(
+        "html_theme", ["--html-theme"],
+        group="output_options",
+        type=str,
+        choices=['green', 'blue'],
```

Comment:
Does it make sense to get this list from CssRenderer.Themes, like `list(CssRenderer.Themes.keys())`

Reply:
That makes sense. But then you should also define the default in the CssRenderer. I prefer `CssRenderer.get_themes()` to get the list of themes and `CssRenderer.get_default_theme()` to get the default. Then the configuration is independent of the data storage in the `CssRenderer`.

Reply:
That sounds even better